shared/instance.go

```diff
@@ -82,6 +82,10 @@ var InstanceConfigKeysAny = map[string]func(value string) error{
 	"boot.stop.priority":         validate.Optional(validate.IsInt64),
 	"boot.host_shutdown_timeout": validate.Optional(validate.IsInt64),
 
+	"cluster.evacuate": validate.Optional(func(value string) error {
+		return validate.IsOneOf(value, []string{"auto", "migrate", "stop"})
+	}),
```

Comment:
Can't that be made into just `"cluster.evacuate": validate.Optional(validate.IsOneOf(value, []string{"auto", "migrate", "stop"})`?

Reply:
Nope, function signatures don't match.

Reply:
Oh right, I'll send a PR to make it work :)